packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -206,4 +208,11 @@ type BaseBoxProps = Omit<BoxProps, keyof BoxVisualProps> &
     }
   >;
 
-export { BaseBoxProps, BoxProps, StyledPropsBlade, validBoxAsValues };
+// ref prop type
+type BoxRefType = Platform.Select<{
+  web: Omit<HTMLDivElement, 'style'>;
```

Comment:
I am removing style from type so that it will atleast warn consumers if they are doing something hacky

Reply:
🤔 hmm okay so Box can also accept "as" prop so you need to ensure the this type will be compatible with that. 

See if just HTMLElement might work or not.

Reply:
yup makes sense. turned it into HTMLElement.